services/ResourceService/src/index.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Islandora\ResourceService;
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+use Silex\Application;
+use Islandora\Chullo\Chullo;
+use Islandora\Chullo\TriplestoreClient;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Psr\Http\Message\ResponseInterface;
+use Silex\Provider\TwigServiceProvider;
+use Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory;
+
+
+date_default_timezone_set('UTC');
+
+$app = new \Silex\Application();
+
+$app['debug'] = true;
+
+$app->register(new \Silex\Provider\TwigServiceProvider(), array(
+  'twig.path' => __DIR__.'/../templates',
+));
+
+$app['twig'] = $app->share($app->extend('twig', function($twig, $app) {
+
+  return $twig;
+}));
+
+$app['fedora'] =  $app->share(function() {
```

Comment:
http://pimple.sensiolabs.org/

Check out the usage section.  If you want a new one per request from the container, you have to use $app->factory().

But now I'm wondering what version of pimple silex uses.